torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -424,7 +426,7 @@ LaunchParams FusionExecutor::computeLaunchParams(
           " to set launch bounds but could not.");
 
       // apply padding to the extent if needed
-      if (warp_padded_extent_set.count(extent)) {
+      if (has_warp_padded_extents && warp_padded_extent_set.count(extent)) {
```

Comment:
Does this really have a non-negligible performance benefit?

Reply:
I checked a few times and confirmed this one didn't do much above the noise level. Good catch :). I can go through the PR again to see if there're other un-necessary ones. 

Contributions from quite a few of these smaller changes tend to be different between profiler and google benchmark runs.

Reply:
Removed this one. Thanks!